src/epd/GxEPD2_213_FC1.cpp

```diff
@@ -327,158 +191,166 @@ void GxEPD2_213_FC1::_PowerOff()
-    _writeCommand(0x82); //vcom_DC setting
-    _writeData (0x12);
-
-    // _writeCommand(0x50); // Some border setting command could be useful later?
-    // _writeData(0x17);
-
-    _writeCommand(0x20);
-    _writeDataPGM(lut_20_vcomDC_partial, sizeof(lut_20_vcomDC_partial), 44 - sizeof(lut_20_vcomDC_partial));
-    _writeCommand(0x21);
-    _writeDataPGM(lut_21_ww_partial, sizeof(lut_21_ww_partial), 42 - sizeof(lut_21_ww_partial));
-    _writeCommand(0x22);
-    _writeDataPGM(lut_22_bw_partial, sizeof(lut_22_bw_partial), 42 - sizeof(lut_22_bw_partial));
-    _writeCommand(0x23);
-    _writeDataPGM(lut_23_wb_partial, sizeof(lut_23_wb_partial), 42 - sizeof(lut_23_wb_partial));
-    _writeCommand(0x24);
-    _writeDataPGM(lut_24_bb_partial, sizeof(lut_24_bb_partial), 42 - sizeof(lut_24_bb_partial));
-  }
-  _PowerOn();
-  _using_partial_mode = true;
+  if (_configured_for_fast) return;  // If already configured, abort
+
+  // Soft reset (via panel setting register)
+  _writeCommand(0x00);
+  _writeData(0); // [0] RST_N
+  _waitWhileBusy("soft reset", 0);
+
+  // Panel setting
+  // [7:6] Display Res, [5] LUT, [4] BW / BWR [3] Scan Vert, [2] Shift Horiz, [1] Booster, [0] !Reset
+  _writeCommand(0x00);
+  _writeData( 0b11 << 6 | 1 << 5 | 1 << 4 | 1 << 3 | 1 << 2 | 1 << 1 | 1 << 0 );
+
+  // VCOM and data interval setting
+  // [7:6] Border, [5:4] Data polarity (default), [3:0] VCOM and Data interval (default)
+  _writeCommand(0x50);
+  _writeData( 0b10 << 6 | 0b01 << 4 | 0b0111 << 0 );
+
+  // Load the various LUTs
+  _writeCommand(0x20);  // VCOM
+  for(uint8_t i=0;i < sizeof(lut_20_vcomDC_partial); i++) 
+    _writeData(pgm_read_byte_near(lut_20_vcomDC_partial + i));
+  
+  _writeCommand(0x21);  // White -> White
+  for(uint8_t i=0;i < sizeof(lut_21_ww_partial); i++) 
+    _writeData(pgm_read_byte_near(lut_21_ww_partial + i));
+  
+  _writeCommand(0x22);  // Black -> White
+  for(uint8_t i=0;i < sizeof(lut_22_bw_partial); i++) 
+    _writeData(pgm_read_byte_near(lut_22_bw_partial + i));
+  
+  _writeCommand(0x23);  // White -> Black
+  for(uint8_t i=0;i < sizeof(lut_23_wb_partial); i++) 
+    _writeData(pgm_read_byte_near(lut_23_wb_partial + i));
+  
+  _writeCommand(0x24);  // Black -> Black
+  for(uint8_t i=0;i < sizeof(lut_24_bb_partial); i++) 
+    _writeData(pgm_read_byte_near(lut_24_bb_partial + i));
+
+  _configured_for_fast = true;
+  _configured_for_full = false;
 }
 
 void GxEPD2_213_FC1::_Update_Full()
 {
+  _Init_Full();
+  _PowerOn();
   _writeCommand(0x12);
   _waitWhileBusy("_Update_Full", full_refresh_time);
   _PowerOff();
 }
 
 void GxEPD2_213_FC1::_Update_Part()
 {
+  _Init_Part();
+  _PowerOn();
   _writeCommand(0x12);
   _waitWhileBusy("_Update_Part", partial_refresh_time);
   _PowerOff();
 }
+
+// Fast refresh waveform is unofficial (experimental?)
+// https://github.com/todd-herbert/heltec-eink-modules/tree/v4.1.2/src/Displays/LCMEN2R13EFC1/LUTs
+
+const unsigned char GxEPD2_213_FC1::lut_20_vcomDC_partial[] PROGMEM =
```

Comment:
Just before opening pull request, I tested the custom LUT by running the Meshtastic carousel (every 20 seconds) for just less than 3 hours.
~500 fast refreshes were performed, between occasional full refreshes (EInkDynamicDisplay). No negative effect was noticed.

The operation is slightly unconventional, with two distinct phases, but it does seem to produce reasonable results. Would be good to get confirmation here too that the operation is appropriate.